src/utils/command-helpers.ts

```diff
@@ -143,7 +151,7 @@ export const getToken = async (tokenFromOptions) => {
   if (tokenFromConfig) {
     return [tokenFromConfig, 'config']
   }
-  return [null, 'not found']
+  return [undefined, 'not found']
```

Comment:
why did this change as well?

Reply:
This was originally intended to satisfy the type requirements of a pair of functions that use `getToken`'s return value. Our updated commit restores the original return value and creates a new variable whose value is set based on `getToken`'s return value to preserve type safety for these function calls.